spec/requests/user_can_get_event_with_all_attributes_spec.rb

```diff
@@ -0,0 +1,40 @@
+describe 'GET/events' do
+  let(:user) { create(:user) }
+  let!(:group) { create(:group) } 
+
+  let(:user_credentials) { user.create_new_auth_token }
+  let(:headers) { { HTTP_ACCEPT: 'application/json' }.merge!(user_credentials) }
+  
+  before do
+    post "/groups/#{group.id}/events", params: { event: 
+                                                {title: 'Craft Academy',
+                                                description: 'Graduation Party',
+                                                location: 'Stockholm',
+                                                date: '2019-12-12',
+                                                time: '2000-01-01T12:12:12.000Z'
+                                                } }, headers: headers
+  
+    get '/events'
```

Comment:
I don’t get this. Why are you making 2 calls. What are you testing here. The POST or the GET request?

Reply:
Basically we are checking GET request here to make sure that user can get all the attributes back.